proto-google-cloud-datacatalog-v1/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>7012</differenceType>
+    <className>com/google/cloud/datacatalog/v1/SearchCatalogRequest*OrBuilder</className>
```

Comment:
This happens for any proto message that adds a field. These `*OrBuilder` classes are meant to be internal. We're ok with any of them adding get/has/contains methods. See and example here: https://github.com/googleapis/java-asset/blob/master/proto-google-cloud-asset-v1/clirr-ignored-differences.xml

Reply:
Done.